syntax = "proto3";

enum OperationType {
  CREATE = 0;
  UPDATE = 1;
  DELETE = 2;
}

message Order {
  optional int32 id = 1;
  optional int32 product_id = 2;
  optional int32 user_id = 3;
  optional int32 quantity = 4;
  optional float amount = 5;
  optional string status = 6;
  optional OperationType operation = 7;
}
